sample/GoogleIO_17/event

```diff
@@ -0,0 +1,68 @@
+{
+    "id": 173, 
+    "name": "Google IO 2017", 
+    "latitude": 0.0, 
+    "longitude": 0.0, 
+    "location_name": "Shoreline Amphitheatre, Mountain View, CA,", 
+    "start_time": "2017-05-17T10:00:00", 
+    "end_time": "2017-05-19T17:00:00", 
+    "timezone": "UTC", 
+    "description": "<p></p>", 
+    "background_image": "/images/background.png.png", 
+    "logo": "/images/logo.png.png", 
```

Comment:
@iamareebjamal actually .png.png is becasue i saved that with png extension myself. will change that.
now coming to whether the nav_header is not being shown i just saw in code we are referring to @mipmap/ic_launcher which is another hardcoding becasue in generator.py we dont load the image in mipmap directory right?